smarts/core/mission_planner.py

```diff
@@ -202,44 +202,49 @@ def _edge_ids(self, pose: Pose, lane_id: str = None):
         return edge_ids
 
     def cut_in_waypoints(self, sim, pose: Pose, vehicle):
+
+        aggressivenes = 5
         radius = self._mission.task.trigger_radius
         neighborhood_vehicles = sim.neighborhood_vehicles_around_vehicle(
-            vehicle=vehicle, radius=radius
+            vehicle=vehicle, radius=80
         )
 
         position = pose.position[:2]
         lane = self._road_network.nearest_lane(position)
 
-        if neighborhood_vehicles:
-            nei_vehicle = neighborhood_vehicles[0]
+        if len(neighborhood_vehicles) == 0:
+            return []
 
-            target_position = nei_vehicle.pose.position[:2]
-            target_lane = self._road_network.nearest_lane(target_position)
+        nei_vehicle = neighborhood_vehicles[0]
```

Comment:
Perhaps call this variable `target_vehicle` in order to be consistent.

Reply:
Thanks,